test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -5611,7 +5621,7 @@ BOOST_AUTO_TEST_CASE(using_interface_complex)
 	char const* text = R"(
 		interface I {
 			event A();
-			function f();
+			function f() returns(bool);
```

Comment:
I would say create a new PR and remove this case here as it is covered by `interface_function_parameters`.